### PR TITLE
Rewords plant/effect descriptions for clarity (#28156)

### LINKED ISSUE #28156: Plant metabolism typo
## Description
The guidebook entries for plant metabolism need a minor cleanup.

**Screenshots**
The grammar sounds a little weird and hard to read, maybe write something like "makes a non-viable plant viable again" or "a plant, which has become unviable due to mutation, viable again". English is not my first language, so if it only sounds wrong to me just ignore this one.
grafik

Two points at the end:
grafik
grafik

### PR DESCRIPTION
**Pull request by taurie**
## About the PR

Reworded and clarified descriptions for phalanximine, robust harvest, and diethylamine.

## Why / Balance

Fixes #28156 

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

- fix: Reworded certain botany descriptions to increase clarity.

### CODE REVIEW

**Comment by slarticodefast** [comment]
Thanks for working on the issue I opened :)
This reads much better than before in my opinion.

You forgot the change for diethylamine I think?

Also the descriptions in effects.ftl don't need a point at the end of the sentence. The guidebook adds it automatically, so right now there are two points: "for each.."

332033628-3d2cb049-a543-452c-b0ef-deea426c8976

In botany.ftl the points are correct.

**Comment by taurie** [comment]
Thank you for your reply, I will address the punctuation issue tonight. Cheers

**Comment by PolterTzi** [comment]
Thanks for cleaning up my stuff.

HandsomeTheEgg has pointed out an issue with some things (like probabilities and maximums) not being displayed in the descriptions: https://github.com/space-wizards/space-station-14/pull/28038#issuecomment-2120653070 .
Are you planning on fixing that as well?
If not, I get it and will try to fix that myself.

File: Resources/Locale/en-US/guidebook/chemistry/effects.ftl [hunk 1]
```diff
@@ -365,9 +365,9 @@ reagent-effect-guidebook-plant-cryoxadone =
 
 reagent-effect-guidebook-plant-phalanximine =
     { $chance ->
-        [1] Makes
-        *[other] make
-    } a plant not viable due to mutation viable again
+        [1] Restores
+        *[other] restore
+    } viability to a plant rendered nonviable by a mutation.
```
Comment (EmoGarbage404) [inline Resources/Locale/en-US/guidebook/chemistry/effects.ftl:370]:
```suggestion
    } viability to a plant rendered nonviable by a mutation
```

**Review by EmoGarbage404 — APPROVED** [review]

**Pull request merged**